### PR TITLE
Clean up ray constructors

### PR DESCRIPTION
**Pull request by hollasch**
Eliminate the defaulted time parameter originally present in the book (though not reflected in the source).

### CODE REVIEW

File: src/TheNextWeek/ray.h [hunk 1]
```diff
@@ -18,12 +18,11 @@ class ray {
   public:
     ray() {}
 
-    ray(const point3& origin, const vec3& direction) : orig(origin), dir(direction), tm(0)
-    {}
+    ray(const point3& origin, const vec3& direction)
+      : orig(origin), dir(direction), tm(0) {}
```
Comment (armansito) [inline src/TheNextWeek/ray.h:22]: nit: I suggest updating the formatting in https://github.com/RayTracing/raytracing.github.io/blob/dev/books/RayTracingTheNextWeek.html#L74 to match the listing.

File: src/TheNextWeek/ray.h [hunk 2]
```diff
@@ -18,12 +18,11 @@ class ray {
...
 
     ray(const point3& origin, const vec3& direction, double time)
-      : orig(origin), dir(direction), tm(time)
-    {}
+      : orig(origin), dir(direction), tm(time) {}
```
Comment (armansito) [inline src/TheNextWeek/ray.h:25]: ditto

**Review by armansito — APPROVED** [review]
Looks good, modulo formatting nits

File: src/TheNextWeek/ray.h (hunk 1 above)
Comment (hollasch) [reply src/TheNextWeek/ray.h:22]: Are you talking about `ray.h`, or all other classes?

Comment (armansito) [reply src/TheNextWeek/ray.h:22]: Oops, I managed to misread the file name. Carry on!

**Pull request merged**